fitlins/workflows/base.py

```diff
@@ -269,11 +277,11 @@ def _get_evs(info):
         (l1_model, plot_corr, [('design_matrix', 'data')]),
         (get_evs, plot_corr, [('out', 'explanatory_variables')]),
```

Comment:
I'm going to push to your branch with a commit blocking all plotting for now. `get_evs` is going to take rethinking, as we'll need to derive the explanatory variables from the contrasts.

Just a heads up.

Reply:
Yep I was thinking the same thing